projects/go-lib/src/public_api.ts

```diff
@@ -36,6 +36,12 @@ export * from './lib/components/go-off-canvas/go-off-canvas.component';
 export * from './lib/components/go-off-canvas/go-off-canvas.module';
 export * from './lib/components/go-off-canvas/go-off-canvas.service';
 
+// Side Nav
+export * from './lib/components/go-side-nav/go-side-nav.module';
```

Comment:
These are good, might want to export `go-side-nav.component` as well.

Reply:
@grahamhency is there a reason to export the component outside of the module?

Reply:
We talked about it offline, but for the sake of documentation...

I could see `GoSideNavComponent` being utilized within another component as maybe a `ViewChild` or `ContentChildren` and to be able to type that correctly we would need access to that component. With a package like this, the best way to expose classes is through the `public_api.ts` file. This allows us to import like:
`import { GoSideNavComponent } from '@tangoe/goponents';`